common/app/views/fragments/newHeader.scala.html

```diff
@@ -79,6 +79,8 @@
                        data-link-name="nav2 : veggie-burger : show"
                        aria-controls="main-menu">
 
+                <div class="new-header__break"></div>
```

Comment:
I'm not really a fan of this. Do you think there could be another solution to it? I'm happy to help you with it, if you like ...

Reply:
I wrote it and I am not a fan either 😆 . If you have time to help think of another solution that would be great!